### PR TITLE
Add more toggles to handle RouteSRV

### PR DESCRIPTION
**Pull request by MustafaSaber**
Signed-off-by: Mustafa Abdelrahman <mustafa.abdelrahman@zalando.de>

### CODE REVIEW

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -172,6 +172,15 @@ skipper_routesrv_cpu: "100m"
 skipper_routesrv_memory: "1Gi"
 {{end}}
 
+skipper_routesrv_node_affinity_enabled: "false"
+skipper_routesrv_min_replicas: 2
```
Comment (szuecs) [inline cluster/config-defaults.yaml]: 1 or at least 1 in test clusters

File: cluster/config-defaults.yaml [hunk 2]
```diff
@@ -172,6 +172,15 @@ skipper_routesrv_cpu: "100m"
...
+skipper_routesrv_max_replicas: 10
+skipper_routesrv_target_average_utilization_cpu: "60"
```
Comment (szuecs) [inline cluster/config-defaults.yaml]: 60->80

File: cluster/config-defaults.yaml [hunk 3]
```diff
@@ -172,6 +172,15 @@ skipper_routesrv_cpu: "100m"
...
+skipper_routesrv_target_average_utilization_memory: "80"
+skipper_ingress_hpa_scale_down_wait: "600"
```
Comment (szuecs) [inline cluster/config-defaults.yaml]: drop that

File: cluster/config-defaults.yaml [hunk 4]
```diff
@@ -172,6 +172,15 @@ skipper_routesrv_cpu: "100m"
...
+skipper_ingress_hpa_scale_up_max_perc: "100"
```
Comment (szuecs) [inline cluster/config-defaults.yaml]: drop that

File: cluster/manifests/skipper/hpa-routesrv.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        kind: ClusterScalingSchedule
+        name: {{ index $name_target 0 }}
+      metric:
+        name: {{ index $name_target 0 }}
+      target:
+        averageValue: {{ index $name_target 1 }}
+        type: AverageValue
+  {{ end }}
+{{ end }}
+  behavior:
```
Comment (szuecs) [inline cluster/manifests/skipper/hpa-routesrv.yaml]: `behavior` should be dropped. It's only important for the data plane.

**Comment by szuecs** [comment]
:+1:

**Comment by MustafaSaber** [comment]
👍

**Pull request merged**